android/src/main/java/com/stripeterminalreactnative/Mappers.kt

```diff
@@ -148,6 +149,25 @@ internal fun mapFromDeviceType(type: DeviceType): String {
     }
 }
 
+internal fun mapToDeviceType(type: String): DeviceType? {
+    return when (type) {
+        "chipper1X" -> DeviceType.CHIPPER_1X
+        "chipper2X" -> DeviceType.CHIPPER_2X
+        "cotsDevice" -> DeviceType.COTS_DEVICE
+        "etna" -> DeviceType.ETNA
+        "stripeM2" -> DeviceType.STRIPE_M2
+        "stripeS700" -> DeviceType.STRIPE_S700
+        "stripeS700Devkit" -> DeviceType.STRIPE_S700_DEVKIT
+        "verifoneP400" -> DeviceType.VERIFONE_P400
+        "wiseCube" -> DeviceType.WISECUBE
+        "wisePad3" -> DeviceType.WISEPAD_3
+        "wisePad3s" -> DeviceType.WISEPAD_3S
+        "wisePosE" -> DeviceType.WISEPOS_E
+        "wisePosEDevkit" -> DeviceType.WISEPOS_E_DEVKIT
+        else -> null
+    }
```

Comment:
How do we keep this list up to date; i.e when there is a new device type on the android sdk, can we have a two way map; i.e also have a `mapFromDeviceType` if one does not exist already, because an exhaustive when with the android `DeviceType` enum model would result in a compilation error when there is a new `DeviceType` enum. 
Also do we have tests for the `Mappers`?

Reply:
yes, there has a `mapFromDeviceType`, I think we need to sync manually when there is a new device type..